backend/src/main/java/org/eclipse/tractusx/puris/backend/common/api/logic/service/VariablesService.java

```diff
@@ -47,8 +48,23 @@ public class VariablesService {
     @Value("${own.bpnl}")
     private String ownBpnl;
 
+    @Value("${own.name}")
```

Comment:
We need some documentation (e.g. in the application.properties or here) how these properties are meant to be used.

Reply:
Done